openapi.yaml

```diff
@@ -478,24 +480,29 @@ definitions:
       id:
         type: integer
         format: int64
-      developerId:
-        $ref: '#/definitions/DeveloperResource/properties/id'
-      name:
-        type: string
-        description: Name of the game
-        example: Eternal Death Slayer 3
-      score:
-        type: number
-        format: float
-        description: Composite review score of this game out of 5
-        example: 4.5
-      releaseDate:
-        type: string
-        format: date
-        description: Date of the games release
-        example: 12/5/1994
-      links:
-        $ref: '#/definitions/SelfLink'
+      type:
+        $ref: '#/definitions/GameType'
+      attributes:
+        type: object
+        properties:
+          developerId:
+            $ref: '#/definitions/DeveloperResource/properties/id'
+          name:
+            type: string
+            description: Name of the game
+            example: Eternal Death Slayer 3
+          score:
+            type: number
+            format: float
+            description: Composite review score of this game out of 5
+            example: 4.5
+          releaseDate:
+            type: string
+            format: date
+            description: Date of the games release
+            example: 12/5/1994
+          links:
+            $ref: '#/definitions/SelfLink'
```

Comment:
I think this `links` needs to be a sibling to `id` and `type` instead of within `attributes`.

Reply:
Fixed this field and added missing Links field to ReviewResource